.github/workflows/publish.yml

```
@@ -0,0 +1,33 @@
+on:
+  push:
+    tags:
+      # expected format: <package>@<version> (example: socket.io@1.2.3)
+      - '**@*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

actionlint (1.7.4)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)